_layouts/about.html

```diff
@@ -30,9 +30,9 @@ <h1 class="c-section-title -light">{{ page.title }}</h1>
           {% for organization in organizations %}
             <div class="m-card--indexes">
               <a href="{{ organization.link }}" target="_blank" class="index-description">
-                <span style="background-image:url('{{site.baseurl}}/img/logos/{{ organization.logo }}')" alt=""></span>
+                <img src="{{site.baseurl}}/img/logos/{{ organization.logo }}" alt="{{ organization.name }} logo">
                 <h3 class="name">{{ organization.name }}</h3>
```

Comment:
Ops... I forgot to tell you. This is an span with a bg image for a reason.
As this images will be updated by the client, we can better control it size/proportion with a background. 
Can you back to the original version?

Sorry!